x-pack/plugins/ingest_manager/server/services/epm/registry/index.ts

```diff
@@ -158,12 +158,12 @@ export function pathParts(path: string): AssetParts {
 
   let [pkgkey, service, type, file] = path.split('/');
 
-  // if it's a dataset
-  if (service === 'dataset') {
+  // if it's a data stream
+  if (service === 'data_stream') {
     // save the dataset name
     dataset = type;
```

Comment:
shouldn't we be calling this dataStream now?

Reply:
this should still actually be dataset as it refers to `data_stream.dataset` i.e. `auth` (as in `system.auth`)